src/oplog/heads.js

```diff
@@ -1,9 +1,3 @@
-/**
```

Comment:
What's the reason for removing this jsdoc?

Reply:
TypeScript gives an error when it is there, saying that modules cannot import from outside the module. I'm not too sure how to fix it...any ideas?

Reply:
It's because it shouldn't be there but a few lines below, just before the `Heads` function, if you move it the error goes away

Reply:
Nice ty. Read my comment below for the problems with generating types in the current state and my proposed solution